runtime/integration-tests/src/generic/utils/mod.rs

```diff
@@ -53,7 +52,12 @@ pub fn give_nft<T: Runtime>(dest: AccountId, (collection_id, item_id): (Collecti
 
 pub fn give_balance<T: Runtime>(dest: AccountId, amount: Balance) {
 	let data = pallet_balances::Account::<T>::get(dest.clone());
-	pallet_balances::Pallet::<T>::set_balance(&dest, data.free + amount);
+	pallet_balances::Pallet::<T>::force_set_balance(
+		RawOrigin::Root.into(),
+		T::Lookup::unlookup(dest),
+		data.free + amount,
+	)
+	.unwrap();
```

Comment:
Probably this is a bit Nit from my side @wischli 

The old `set_balance()` has been mapped to `force_set_balance()`. I feel like if any state change in the runtime is done through "extrinsic" calls, it is easier to replicate later in development for the Apps team. They would only need to go through `state_mut` or `sumbit()` to know which extrinsic they should call to replicate the same actions